src/objects.rs

```diff
@@ -1277,9 +1282,81 @@ pub struct ODRule {
     pub network_id: Option<String>,
     pub physical_mode_id: Option<String>,
 }
+impl_id!(ODRuleV1);
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[cfg(feature = "fare-v2")]
+pub struct Ticket {
+    pub id: String,
+    pub name: String,
+    pub comment: String,
+}
+#[cfg(feature = "fare-v2")]
 impl_id!(Ticket);
-impl_id!(ODRule);
+
+#[cfg(feature = "fare-v2")]
+impl GetObjectType for Ticket {
+    fn get_object_type() -> ObjectType {
+        ObjectType::Ticket
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[cfg(feature = "fare-v2")]
+pub struct TicketPrice {
+    pub ticket_id: String,
+    pub price: f64, // TODO: how to ensure this value is always positive (u64 on cents? Setter?)
+    pub currency: String, //TODO: evaluate crates 'iso4217' and 'steel-cent' for encoding currency
```

Comment:
Are we interested by exploring some crate to handle the ISO-4217 for us. I've found these 2 [`iso4217`](https://crates.io/crates/iso4217) and [`steel-cent`](https://crates.io/crates/steel-cent) but they don't seem to have a lot of traction or support.

Reply:
it could be good for validation

Reply:
Done in ae2ac8f803cf9ae94a798d6de59cd0716ea4e19b.

Reply:
I'll try to add some unit test for the custom serializer and deserializer I added.